### PR TITLE
Update eloston-chromium to 58.0.3029.110-1

### PR DESCRIPTION
**Pull request by ghost**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256),
      provide public confirmation by the developer: {{link}}

### CODE REVIEW

**Comment by commitay** [comment]
Ping @vitorgalvao 

Not sure if changing to a newer fork is okay?

**Comment by commitay** [comment]
Closing in favour of https://github.com/caskroom/homebrew-cask/pull/35817, Github won't let me edit this PR.

**Comment by vitorgalvao** [comment]
> Not sure if changing to a newer fork is okay?

It’s only OK if the new fork is the new official one, as stated by the official sources. Otherwise it’s not OK.

> Github won't let me edit this PR.

Seems like @wickedsp1d3r deleted the branch (see `unknown repository` at the top), so Github has no source to edit.

**Comment by commitay** [comment]
> It’s only OK if the new fork is the new official one, as stated by the official sources. Otherwise it’s not OK.

Cool, I don't know if it happens very often but I'll remember that for future reference.

> deleted the branch (see unknown repository at the top), so Github has no source to edit.

Thanks, I didn't even notice that.

**Comment by vitorgalvao** [comment]
> I don't know if it happens very often

Not often, no. But a huge chunk of occurrences are users that grabbed a (semi-)popular piece of software that is no longer under development (though not necessarily officially abandoned), forked it, made a few superficial changes, and even though their fork has 5 stars and the original repo has 800, they try to change the cask to their fork.

Those are huge no-nos, as you might imagine. Nothing guarantees us their changes are good, that they’re trustworthy, or that the community would prefer their fork over something else (including the abandoned source).

However, if a software package states right there in the README “this is abandoned; you should use fork X”, and link to it, then the change can occur even in the 5/800 stars case, because _the original developer is explicitly endorsing the fork as the new canonical_.

Bottom line that’s what you have to worry about: “is this source the official one?”.

**Comment by commitay** [comment]
Thank you for the thorough response, I'll add a bullet point to "rejected casks" that states forks must be offical/endorsed and link to your comment. I'm taking note of the questions I've asked (and will be asking) to include in a future PR to expand the "maintaining" doc and I'll make sure to include something that covers "changing sources".
